spring-boot-project/spring-boot-docs/src/docs/antora/modules/appendix/pages/application-properties/index.adoc

```diff
@@ -11,39 +11,38 @@ Make sure to review xref:reference:features/external-config.adoc#features.extern
 NOTE: Property contributions can come from additional jar files on your classpath, so you should not consider this an exhaustive list.
 Also, you can define your own properties.
 
-
-include::partial$configuration-properties/core.adoc[]
+include::partial$configuration-properties/actuator.adoc[]
```

Comment:
This commit shouldn't be part of the PR.

Reply:
My mistake, I dropped it.